docs/user/whats-new.asciidoc

```diff
@@ -1,175 +1,144 @@
+==== Manage dashboards more easily and efficiently
+As part of a series of improvements to help you find and manage your dashboards [started in version 8.15], we are adding a way to sort your dashboards by recently viewed (new default), an option to star your favorite dashboards, as well as some statistics to monitor the usage of your dashboards. 
 
-image::https://images.contentstack.io/v3/assets/bltefdd0b53724fa2ce/blt9bc52ff7851acc52/669a4f6a490fbc64fa22f279/field-statistics.gif[Showing field statistics panel in Dashboards]
+You can find your favorite dashboards in the new **Starred** tab.
 
-[discrete]
-==== Statistics in legends
+image::images/dashboard-star.png[Viewing starred dashboards]
 
-Accelerate time to insights by summarizing the values of your charts using average, minimum, maximum, median, and variance, among many others. You can add these statistics for **Lens** and ES|QL visualizations. It is important to note that these statistics are computed using the data points from the chart considering the aggregation used and not the raw data. In the following example, the chart shows the median memory per host, so the Max = 15.3KB for the first series (artifacts.elastic.co) is the maximum value of the median memory per host. 
+By opening a dashboard's details using the “info” icon from the dashboard list view, you can now get a sense of the popularity of that dashboard with a histogram showing how many times the dashboard was viewed in the last 90 days.
 
-image::images/statistics-in-legends.png[Statistics in legends]
+image::images/dashboard-usage.png[Dashboard usage chart]
 
-You can find the option to select statistics for your legends along with an explanation for each calculation when editing your visualization, as shown in the following image.
+[discrete]
+==== Log Pattern Analysis dashboard panels
+Log Pattern Analysis panels are now available for you to add to your dashboards, making AIOps even more embedded in your workflows and where you need it. Filtering patterns adjusts the dashboard’s data accordingly. You can also choose the filtering to transition you into Discover for further exploration. 
 
-image::images/statistics-in-legends2.png[Select statistics in legends]
+image:https://images.contentstack.io/v3/assets/bltefdd0b53724fa2ce/blt8288e01386b5830c/67222fb0d2da223e27bc1e67/log_analysis_panel.gif[Log pattern analysis panel in dashboards]
 
 
 [discrete]
-==== View dashboard creator and last editor
+==== Color text values in tables
+Before, you could only decide to color numeric values in tables. We're adding the ability to also color your string values. You can decide whether you want to color the whole cell, or only the text.
```

Comment:
We could say "Previously" instead of "Before"

Reply:
```suggestion
Previously, you could only decide to color numeric values in tables. We're adding the ability to also color your string values. You can decide whether you want to color the whole cell, or only the text.
```
Some additional comment